package.json

```
@@ -19,17 +19,22 @@
   },
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
+  "bin": {
+    "@antfu/eslint-config": "./bin/index.js",
+    "init": "./bin/index.js"
+  },
```

```suggestion
  "bin": "./bin/index.js",
```

I would prefer a single entry